_commands/tools/ip.md

```diff
@@ -0,0 +1,91 @@
+---
+---
+ip
+----
+
+`ip` is a powerful networking tool that when simply used will display information on kernel-resident interfaces. Much like `ifconfig`, it can turn devices on/off and set IP and MAC addresses. `ip` also wraps popular commands like netstat, ndp, arp, and route into its functionality.
```

Comment:
Could you be a little more consistent with what's code and what's not? e.g. the commands in the last sentence.